pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -251,21 +272,26 @@ def connection_select(context, name):
                      default='all'))
 @add_options(verify_option)
 @click.pass_obj
-def connection_add(context, **options):
+def connection_add(context, name, **options):
     """
     Add a new WBEM connection definition from specified options.
 
     Create a new WBEM connection definition in the connections file from
     the specified options. A connection definition with that name must not
     yet exist.
 
-    The current connection remains unchanged by this command.
+    The NAME argument MUST exist. It define the server uri
+    and the unique name under which this server connection information
+    will be stored. A --server or --mock-server must exist because they
+    define the server. All other properties are optional.
 
-    Examples:
+    Adding a connection does not set the new connection as the current
+    connection. Use `connection select` to set a particular stored connection
+    definition as the current connection.
 
       pywbemcli --name newsrv connection add --server https://srv1
```

Comment:
Update the example to use NAME, e.g.:

pywbemcli connection add newsrv --server https://srv1

Reply:
DONE